sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -79,6 +79,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/core-auth": "^1.1.3",
+    "@azure/identity": "^1.1.0",
```

Comment:
I believe identity is used in samples only?

Reply:
```
$ git grep identity sdk/formrecognizer/ai-form-recognizer/src
sdk/formrecognizer/ai-form-recognizer/src/formRecognizerClient.ts:import { TokenCredential } from "@azure/identity";
sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts:import { TokenCredential } from "@azure/identity";
```

@witemple-msft I recommend to use `core-auth` instead of `identity` for importing `TokenCredential`. For the purposes of this PR, I will keep my change.

Reply:
Should we log a follow up issue to use core-auth in deps and identity in dev deps?

Reply:
Nevermind, found a few mins on my hand, #13677

Reply:
Thanks @ramya-rao-a! can you guys please approve this PR?

Reply:
I see you already merged #13677 first, so I reverted that change here.